app/helpers/devise_helper.rb

```diff
@@ -0,0 +1,16 @@
+module DeviseHelper
+  def devise_error_messages!
+    return "" if resource.errors.empty?
+
+    messages = resource.errors.full_messages.map { |msg| content_tag(:li, msg) }.join
+
+    html = <<-HTML
+    <div class="alert alert-danger">
+      <button type="button" class="close" data-dismiss="alert">x</button>
+      #{messages}
+    </div>
+    HTML
+
+    html.html_safe
+  end
+end
```

Comment:
Get rid of this helper and use the notification plugin I'm using when displaying `flash` messages. Let's be consistent.